src/jobs/deploy_env.yml

```diff
@@ -42,10 +42,6 @@ parameters:
     type: string
     default: PROJECT_NAME
     description: The Deployment resource's name in Kubernetes to interrogate for the previous deployment's version
-  fixed_previous_version:
-    type: string
-    default: ""
-    description: (Only used for orb integration testing) Instead of reading from Kubernetes deployment, provide a fixed APP_VERSION for the previous deployment
```

Comment:
Apparently this was needed for some testing? https://github.com/ministryofjustice/hmpps-circleci-orb/pull/52#discussion_r658587654

Reply:
no longer - I've just made the changelog script play nice if there is no existing deployment found.